src/TypeScript/NodeJSLocator/CompatibleNodeLocator.cs

```diff
@@ -46,7 +39,7 @@ public CompatibleNodeLocator(INodeVersionInfoProvider nodeVersionInfoProvider, I
             this.logger = logger;
         }
 
-        public string Locate()
+        public NodeVersionInfo Locate()
```

Comment:
Signature changed so it could be used in telemetry. Was previously returning only the path, now it returns also the version.